torchvision/transforms/v2/_color.py

```diff
@@ -228,19 +228,22 @@ def __init__(
 
     def _get_params(self, flat_inputs: List[Any]) -> Dict[str, Any]:
         num_channels, *_ = query_chw(flat_inputs)
-        return dict(
-            zip(
-                ["brightness", "contrast1", "saturation", "hue", "contrast2"],
```

Comment:
I'm not sure why we went for `"contrast1"` and `"contrast2"` before. Since we have the additional `"contrast_before"` flag, it is guaranteed, that they will never be applied at the same time. Meaning, there is no point in sampling them twice.